### PR TITLE
Improve model performance

### PR DESCRIPTION
**Pull request by microbit-robert**
The changes made to improve model consistency were never ported over to this version.

I've also added `shuffle` to shuffle the training data before each epoch and set the proportion of data used for validation to zero. We don't do anything with the metrics we get from this validation, so we might as well train on all of the data.

I think the model is otherwise good. So the only thing perhaps left to play with is the number of units in the first dense layer. Edit: Changing the number of units in the first dense layer and changing the batch size in the fit function don't lead to noticable improvements or otherwise. From what I've seen elsewhere, these are sensible values so worth sticking with.

### CODE REVIEW

**Comment by microbit-robert** [comment]
Leaving in draft until we have built enough confidence that this is an improvement to all proposed and existing projects.

**Review by microbit-grace — APPROVED** [review]
LGTM. Tested it with jump, still and walk with the micro:bit strapped on to my ankles and the estimated actions were much more consistent after re-testing.
